packages/dds/quorum/.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+    "extends": [
+        "@fluidframework/eslint-config-fluid"
```

Comment:
Done in latest.  I also removed the explicit-member-accessibility overrides in the strict ruleset -- I think we should mandate in all cases (esp. since we use a mixture of public and private constructors across our repo).